modules/AddBook.js

```diff
@@ -0,0 +1,35 @@
+import Book from './Book.js';
+
+const inputTitle = document.querySelector('#title');
+const inputAuthor = document.querySelector('#author');
+const indicator = document.querySelector('#indicator');
+
+function checkRepetition(book) {
+  for (let i = 0; i < Book.collection.length; i += 1) {
+    const currentBook = Book.collection[i];
+    if (currentBook.title.toLowerCase() === book.title.toLowerCase()
+            && currentBook.author.toLowerCase() === book.author.toLowerCase()) {
+      indicator.textContent = 'Book already added';
+      return false;
+    }
+  }
+  return true;
+}
```

Comment:
- [ ] Kindly change all the function declaration into function expression `arrow function`